content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -282,6 +289,141 @@ and [external.metrics.k8s.io](https://github.com/kubernetes/community/blob/maste
+
+Starting from
+[v1.17](https://github.com/kubernetes/enhancements/blob/master/keps/sig-autoscaling/20190307-configurable-scale-velocity-for-hpa.md)
+the `v2beta2` API allows scaling behavior to be configured through the HPA
+`behavior` field. Behaviors are specified separately for scaling up and down in
+`scaleUp` or `scaleDown` section under the `behavior` field. A stabilization
+window can be specified for both directions which prevents the flapping of the
+number of the replicas in the scaling target. Similarly specifing scaling
+policies controls the rate of change of replicas while scaling.
+
+### Scaling Policies
+
+One or more scaling policies can be specified in the `behavior` section of the spec.
+When multiple policies are specified the policy which allows the highest amount of
+change is the policy which is selected. The following example shows this behavior
+while scaling down:
+
+```yaml
+behavior:
+  scaleDown:
+    policies:
+    - type: Pods
+      value: 4
+      periodSeconds: 60
+    - type: Percent
+      value: 10
+      periodSeconds: 60
+```
+
+When the number of pods is more than 40 the second policy will be used for scaling down.
+For instance if there are 80 replicas and the target has to be scaled down to 10 replicas
+then during the first step 8 replicas will be reduced. When the number of replicas falls
+below 40 the first policy is applied and 4 replicas will be reduced at a time.
+
+The policy selection can be changed by specifying the `selectPolicy` field for a scaling
+direction. By setting the value to `Min` which would select the policy which allows the
+smallest change in the replica count. Setting the value to `Disabled` completely disabled
+scaling in that direction.
+
+### Stabilization Window
+
+The stabilization window is used to retrict the flapping of replicas when the metrics
+used for scaling keep fluctuating. The stabilization window is used by the autoscaling
+algorithm to consider the computed desired state from the past to prevent scaling. In
+the following example the stabilization window is specified for `scaleDown`.
+
+```yaml
+scaleDown:
+  stabilizationWindowSeconds: 300
+```
+
+When the metrics indicate that the target should be scaled down the algorithm looks
+into previously computed desired states and uses the highest value from the specified
+interval. In above example all desired states from the past 5 minutes will be considered.
+
+### Default Behavior
+
+To use the custom scaling not all fields have to be specified. Only values which need to be
+customized can be specified. These custom values are merged with default values. The defaul values
+match the existing behavior in the HPA algorithm. This includes a 5 minute scale down stabilization window (or
+the value of the `--horizontal-pod-autoscaler-downscale-stabilization` flag if
+provided) and the maximum of 4 pods and 2x scale up limit.
+
+```yaml
+behavior:
+  scaleDown:
+    stabilizationWindowSeconds: 300
```

Comment:
We have the same default window length for scaling up? Just curious.
It'd be desirable to spell out what EXACTLY the default behavior is so users know what can/should be customized.

Reply:
I've described the defaults in some more detail